src/fides/api/ops/api/v1/scope_registry.py

```diff
@@ -169,18 +219,47 @@
     CONNECTION_AUTHORIZE: "OAuth2 Authorization",
     CONNECTION_TYPE_READ: "View types of connections",
     CONSENT_READ: "Read consent preferences",
+    CTL_DATASET_CREATE: "Create a ctl dataset",
+    CTL_DATASET_READ: "Read ctl datasets",
+    CTL_DATASET_DELETE: "Delete a ctl dataset",
+    CTL_DATASET_UPDATE: "Update ctl datasets",
+    CTL_POLICY_CREATE: "Create a ctl policy",
+    CTL_POLICY_READ: "Read ctl policies",
+    CTL_POLICY_DELETE: "Delete a ctl policy",
+    CTL_POLICY_UPDATE: "Update ctl policies",
     DATABASE_RESET: "Reset the application database",
+    DATA_CATEGORY_CREATE: "Create a data category",
```

Comment:
..Breaking CTL-scopes into individual scopes per endpoint so we can be more targeted about which specific permissions are granted to various roles and system managers.

Reply:
Thank you for cleaning up my mess 🙏